src/installer/managed/Microsoft.NET.HostModel/Bundle/Bundler.cs

```diff
@@ -346,33 +347,16 @@ public string GenerateBundle(IReadOnlyList<FileSpec> fileSpecs)
             HostWriter.SetAsBundle(bundlePath, headerOffset);
 
             // Sign the bundle if requested
-            if (_macosCodesign && RuntimeInformation.IsOSPlatform(OSPlatform.OSX) && HostModelUtils.IsCodesignAvailable())
+            if (_macosCodesign && RuntimeInformation.IsOSPlatform(OSPlatform.OSX) && Codesign.IsAvailable)
             {
-                var (exitCode, stdErr) = HostModelUtils.RunCodesign("-s -", bundlePath);
+                var (exitCode, stdErr) = Codesign.Run("-s -", bundlePath);
```

Comment:
The bundler did something that didn't work well with the other signing implementation, so I left it to use `codesign`. I think it would work with this, but just haven't tried it and was going to do that in a follow up. I can try it and add it to this PR if you'd prefer, though.